projects/fusion-ui/components/chart-labels/v4/chart-labels-v4.component.html

```diff
@@ -9,7 +9,8 @@
         >
         </div>
         <fusion-icon class="fu-chart-label-icon" *ngIf="datasetLabel.icon" [name]="datasetLabel.icon"></fusion-icon>
-        <div class="fu-chart-label-text truncate" [attr.data-testid]="testId"
+        <div class="fu-chart-label-text truncate"
+             [attr.data-testid]="datasetLabel.label === 'Unity Ads' ? testId : null"
```

Comment:
Please don't use something related to the real texts or mocks
like "datasetLabel.label === 'Unity Ads'"